Makefile

```diff
@@ -10,7 +10,16 @@ generate:
 	goimports -w -local kubevirt.io cmd/ pkg/ tests/
 	./hack/bootstrap-ginkgo.sh
 
-build: checksync fmt vet compile
+docker-builder-create:
+	./hack/docker-builder.sh create
+
+docker-builder-clean:
+	./hack/docker-builder.sh clean
+
+build: checksync fmt vet docker-builder-create
+	./hack/docker-builder.sh build ${WHAT}
+
+obsolete-build: checksync fmt vet compile
```

Comment:
Maybe still calling it build - but raising an warning (echo … >&2) that there might be side effects if a user calls it directly?